compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -1264,6 +1264,7 @@ void quantize_const_inputs(luci::CircleNode *node, loco::DataType output_type)
     case luci::CircleOpcode::EXP:
     case luci::CircleOpcode::FLOOR:
     case luci::CircleOpcode::FLOOR_DIV:
+    case luci::CircleOpcode::GATHER:
```

Comment:
Here is all const input quantization case.
If we don't need to quantize indices, you can add gather to above case. (Line 1250 case)

Reply:
Ah, got it! thanks!